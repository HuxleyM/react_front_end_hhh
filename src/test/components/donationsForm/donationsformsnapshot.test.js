import React, { Component } from 'React';
import { create } from "react-test-renderer";
import DonationForm from '../../../components/donationsForm'


describe("Donations Form", () => {
  test("it matches the snapshot", () => {
    const component = create(<DonationForm />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
