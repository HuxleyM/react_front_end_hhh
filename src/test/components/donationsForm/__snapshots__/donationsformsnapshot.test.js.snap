// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Donations Form it matches the snapshot 1`] = `
<form>
  <label>
    Amount
  </label>
  <input
    id="amount"
    placeholder="amount"
    type="text"
  />
  <input
    id="passphrase"
    placeholder="rhinounicorn"
    type="text"
  />
  <input
    id="submit_form"
    type="submit"
    value="Submit"
  />
</form>
`;
